970-ng-commons/contracts/openapi/v1/openapi.yaml

```
@@ -7,70 +7,118 @@ info:
+    | order      | string  | desc    | Order to sort on. Type: enum(asc,desc)                                                                                                                                     |
+    | sort       | string  | none    | The field to sort against. Note: Specify the fields which are to be used for sorting. Thought should be given to operational overhead of sort fields. |
+    | limit      | int     | 30      | Pagination: Number of items to return                                                                                                                 |
+    | page       | int     | 1       | Pagination page number strategy: Specify the page number within the paginated collection related to the number of items in each page                  |
+    | created    | int64   | none    | Unix timestamp when the resource was created in milliseconds                                                               |
+    | updated    | int64   | none    | Unix timestamp when the resource was last edited in milliseconds                                                           |
     
-    ## Versioning
     
-      The current version of our API is **1.0**.
+    ## Status Codes
+
+      Harness follows conventional HTTP status codes to indicate status of the API request. 
+      Generally, 2xx responses are reserved for a success, while 4xx status codes are reserved for failures. 5xx response implies a server error on Harness' side.
+
+      | Error Code  | Description |
+      |-------------|-------------|
+      | 200         |     OK      |
+      | 201         |   Created   |
+      | 202         |   Accepted  |
+      | 204         |  No Content |
+      | 400         | Bad Request |
+      | 401         | Unauthorized |
+      | 403         | Forbidden |
+      | 412         | Precondition Failed |
+      | 415         | Unsupported Media Type |
+      | 500         | Server Error |
     
-      The version number represents the core API and does not change frequently. The version number changes only if there is a significant departure from the basic underpinnings of the existing API. For example, the version number would change with a system-wide refactoring of core concepts or resources.
-      This version number is represented within the API path. For example, the v1 in `api.harness.io/v1/` is the version number.
+      To view our Error Responses structures, please click [here](#tag/Error-Response).
+
+    ## Versioning
+
+    ### Harness Version
+      The current version of our Beta APIs is **to be announced**. This will be a Date Header format, and will be valid only for our Beta APIs.
     
+    ### Generation
+      All our Beta APIs have a **Generation** that is represented within API path. For example, the v1 in `app.harness.io/v1/`.
```

```suggestion
      All our beta APIs are versioned, and the version is included in the path to every API resource. For example, v1 beta APIs begin with `app.harness.io/v1/`, where v1 is the API version.
```
* Is that a generation or a version? I'm assuming version. Let me know if generation is correct (and if generation is correct, replace version in my suggestion.)